stable/kube-bench/templates/cron.yaml

```
@@ -15,41 +17,23 @@ spec:
           hostPID: true
           restartPolicy: Never
           containers:
-            - name: kube-bench
+            - name: {{ .Values.jobName }}
```

Here also the convention is:

```suggestion
            - name: {{ .Chart.Name }}
```

---

added :)